packages/input-number/src/input-number.vue

```diff
@@ -13,7 +13,7 @@
       role="button"
       :title="t('el.spinner.decrease')"
       v-if="controls"
-      v-repeat-click="decrease"
+      v-repeat-click:[sensitivity]="decrease"
```

Comment:
This syntax (a dynamic "argument" to the directive) is advertised in the Vue.js documentation: https://vuejs.org/v2/guide/custom-directive.html#Dynamic-Directive-Arguments

Reply:
Cool. I was not aware of this feature.